src/reql/tree.rs

```diff
@@ -1,5 +1,6 @@
+use protobuf::ProtobufEnum;
 use rustc_serialize::json::{Json, ToJson};
-use super::ql2::Term_TermType;
+use super::super::ql2::Term_TermType;
```

Comment:
True.  I don't like this but need to figure out what the right way around it is.